<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCMapTool</class>
 <widget class="QWidget" name="QGCMapTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="100,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGCMapWidget" name="map" native="true">
     <widget class="QGCMapRing" name="Ring" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>331</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_x">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-200.000000000000000</double>
      </property>
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_y">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-200.000000000000000</double>
      </property>
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_x">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_y">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>60</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>y:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QSlider" name="zoomSlider">
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGCMapToolBar" name="toolBar" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGCMapToolBar</class>
   <extends>QWidget</extends>
   <header>QGCMapToolBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QGCMapWidget</class>
   <extends>QWidget</extends>
   <header>QGCMapWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QGCMapRing</class>
   <extends>QWidget</extends>
   <header>QGCMapRing.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
